javascript/ql/src/Security/CWE-116/IncompleteSanitization.qhelp

```diff
@@ -31,6 +31,14 @@ still have undesirable effects, such as badly rendered or confusing output.
 Use a (well-tested) sanitization library if at all possible. These libraries are much more
 likely to handle corner cases correctly than a custom implementation.
 </p>
+
+<p>
+
+An even safer alternative is to design the application such that sanitization isn't
+needed at all, for instance by using prepared statements for SQL queries.
```

Comment:
```suggestion
An even safer alternative is to design the application so that sanitization is not
needed, for instance by using prepared statements for SQL queries.
```